src/DevTools/__test__/MockRelayRenderer.test.tsx

```diff
@@ -34,4 +39,36 @@ describe("MockRelayRenderer", () => {
       done()
     }, 10)
   })
+
+  it("renders an error when child components throw", done => {
+    console.log = () => null // MockRelayRenderer prints out error info to the console, let's silence it.
+    const tree = mount(
+      <MockRelayRenderer
+        Component={Artwork}
+        query={badQuery}
+        mockResolvers={{
+          Artwork: () => ({
+            title: "Mona Lisa",
+            image: {
+              url: "http://test/image.jpg",
+            },
+            artist: null,
+          }),
+        }}
+      />
+    )
+    tree.setState({
+      caughtError: {
+        error: new Error("Hey it's an error!"),
+        errorInfo: {},
+      },
+    })
+    setTimeout(() => {
+      console.log(tree.text())
+      expect(tree.update().text()).toEqual(
+        "Error occurred while rendering Relay component: Error: Hey it's an error!"
+      )
+      done()
+    }, 10)
```

Comment:
Instead of this timeout you could also use the renderUntil utility.

Reply:
Ah, yes! Good point. 

I played around with a bit and was initially flummoxed by some React/Enzyme lifetime stuff around the initial state of the component I set, but then I realized that I don't need the timeout at all 😅 I must've copied it from the other test, sorry about that! I've updated the other test to use `renderUntil`.